test/sharness/lib/test-lib.sh

```diff
@@ -409,3 +430,9 @@ convert_tcp_maddr() {
 port_from_maddr() {
   echo $1 | awk -F'/' '{ print $NF }'
 }
+
+
+fail() {
```

Comment:
There is `test_fsh` - https://github.com/ipfs/go-ipfs/blob/1054826ac49f66d329d3088143f1a7f2b9930dcc/test/ipfs-test-lib.sh#L22-L27 and it's used all over sharness tests - like `test_cmp expected unique_hash || test_fsh cat add_output`

Reply:
Ah nice!